main.go

```diff
@@ -183,6 +183,7 @@ func main() {
 	http.HandleFunc(util.GetFullPath("api/version"), getVersion)
 	http.HandleFunc(util.GetFullPath("api/user"), api.GetUser)
 	http.HandleFunc(util.GetFullPath("api/states"), handleWithDB(api.ListStates, database))
+	http.HandleFunc(util.GetFullPath("api/states_lineages"), handleWithDB(api.ListStatesWithLineages, database))
```

Comment:
Why not `api/lineages` directly?

In fact, I'd use `api/lineages/<lineage>/states` to list states by lineage

Reply:
This endpoint is a little bit particular and it's not aimed to recover all states of a lineage
His purpose is to return a list of all state's paths with the associated lineage in order to fill the nav select since we need lineage to access to state view (`/lineage/<lineage>?versionid=<versionid>`)

Reply:
Hmmmm I think we'll need to talk about that. I'm not sure to understand.

Reply:
Maybe it's me who explains very badly ^^ 

This is a new endpoint which return something like this:
```json
[
    {
        "path": "terraform_0.12.28.tfstate",
        "lineage": "b7d2aa5a-812a-2d1f-e5e6-835215928e00",
        "version_id": "1234"
    },
    {
        "path": "terraform_0.13.5.tfstate",
        "lineage": "b7d2aa5a-812a-2d1f-e5e6-835215928e00",
        "version_id": "5678"
    },
    ...
]
```

And I'm using that to populate the state quick access (top-right select element on Terraboard).
I put the path as item label and the whole object as value to be able to build the correct URL when a user select a path to view it.

Reply:
OK. How about `/api/lineages/last`?

Reply:
It doesn't seem very relevant to me since this endpoint returns all the states and not just the last ones.
The drop-down menu that it populates is really intended to list all the path existing in the database (without duplicates however) to allow the user to quickly access a specific state file now that we only list the last by lineage on the overview.

Reply:
Let's only list lineages and ignore potentially renamed paths.